.github/workflows/lint.Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.10-alpine
+
+WORKDIR /app
+
+RUN apk update && apk add git freetype-dev gcc musl-dev libffi-dev g++ curl tar python3
+
+RUN adduser --disabled-password prospector prospector \
+    && chown -R prospector:prospector /app \
+    && rm -rf ${HOME}/.cache/ ${HOME}/.local/bin/__pycache__/
+USER prospector
+
+ENV PATH /home/prospector/.local/bin:${PATH}
+
+
+COPY requirements.txt /app/
+run pip3 install prospector
```

Comment:
This "run" command should be capitalized: "RUN"

Reply:
Added this in the newest version